roles/ComputeHCISriov.yaml

```diff
@@ -0,0 +1,68 @@
+###############################################################################
+# Role: ComputeHCISriov                                                            #
+###############################################################################
+- name: ComputeHCISriov
+  description: |
+    Compute Node with SR-IOV role hosting Ceph OSD too
+  networks:
+    InternalApi:
+      subnet: internal_api_subnet
+    Tenant:
+      subnet: tenant_subnet
+    Storage:
+      subnet: storage_subnet
+    StorageMgmt:
+      subnet: storage_mgmt_subnet
+  RoleParametersDefault:
+    TunedProfileName: "cpu-partitioning"
```

Comment:
Different from ComputeHCI tuned profile, in this case It has to be cpu-partitioning because of SR-IOV.